charts/sysdig-deploy/templates/_helpers.tpl

```diff
@@ -0,0 +1,37 @@
+{{/*
+Determine sysdig monitor endpoint based on provided region
+*/}}
+{{- define "global.monitorUrl" -}}
```

Comment:
I think the `global` here and in the helper below is misleading, just `monitorUrl` is probably fine